test/VS.Web.CG.EFCore.Test/TestModels/TestModel.cs

```diff
@@ -19,7 +19,9 @@ public static IModel CategoryProductModel
         {
             get
             {
+#pragma warning disable EF1001 // Internal EF Core API usage.
                 var builder = new ModelBuilder(SqlServerConventionSetBuilder.Build());
```

Comment:
What is the EF internal code that is being used here? As far as I can tell, everything here is now public.

Reply:
SqlServerConventionSetBuilder.Build() is marked as [EntityFrameworkInternal]

Reply:
Ah, found the issue. These are public in master, but that that hasn't propagated to the release yet. So you should be able to remove these pragmas before RTM.